tests/adapters_t/batchsystems_t/test_slurm.py

```diff
@@ -0,0 +1,205 @@
+        cls.mock_config = cls.mock_config_patcher.start()
+        cls.mock_async_run_command = cls.mock_async_run_command_patcher.start()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.mock_config_patcher.stop()
+        cls.mock_async_run_command_patcher.stop()
+
+    def setUp(self):
+        self.cpu_ratio = 0.5
+        self.memory_ratio = 0.25
+
+        self.command = 'sinfo --Format="statelong,cpusstate,allocmem,memory,features,nodehost" -e --noheader -r --partition=test_part'
+        #  self.command = 'sinfo --Format="statelong,cpusstate,allocmem,memory,features,nodehost" -e --noheader'
+
+        self.command_wo_options = 'sinfo --Format="statelong,cpusstate,allocmem,memory,features,nodehost" -e --noheader -r'
+        #  self.command_wo_options = 'sinfo --Format="statelong,cpusstate,allocmem,memory,features,nodehost" -e --noheader'
+
+        return_value = "\n".join(
+            [
+                "mixed      2/2/0/4   6000    24000   VM-1   host-10-18-1-1",
+                "mixed      3/1/0/4   15853   22011   VM-2   host-10-18-1-2",
+                "mixed      1/3/0/4   18268   22011   VM-3   host-10-18-1-4",
+                "mixed      3/1/0/4   17803   22011   VM-4   host-10-18-1-7",
+                "draining   0/4/0/4   17803   22011   draining_machine   draining_machine",
+                "idle       0/4/0/4   17803   22011   idle_machine   idle_machine",
+                "drained    0/4/0/4   17803   22011   drained_machine   drained_machine",
+                "powerup    0/4/0/4   17803   22011   power_up_machine   power_up_machine",
+            ]
+        )
+
+        self.mock_async_run_command.return_value = async_return(
+            return_value=return_value
+        )
+
+        self.setup_config_mock(
+            options=AttributeDict({"long": {"partition": "test_part"}})
+        )
+
+        self.slurm_adapter = SlurmAdapter()
+
+    def tearDown(self):
+        self.mock_async_run_command.reset_mock()
+
+    def setup_config_mock(self, options=None):
+        self.config = self.mock_config.return_value
+        self.config.BatchSystem.max_age = 10
+        if options:
+            self.config.BatchSystem.options = options
+        else:
+            self.config.BatchSystem.options = {}
+
+    def test_disintegrate_machine(self):
+        self.assertIsNone(
+            run_async(self.slurm_adapter.disintegrate_machine, drone_uuid="test")
+        )
+
+    def test_drain_machine(self):
+        run_async(self.slurm_adapter.drain_machine, drone_uuid="VM-1")
+        self.mock_async_run_command.assert_called_with(
+            "scontrol update NodeName=host-10-18-1-1 State=DRAIN Reason='COBalD/TARDIS'"
+        )
+
+        self.mock_async_run_command.reset_mock()
+
+        run_async(self.slurm_adapter.drain_machine, drone_uuid="VM-1")
+        self.mock_async_run_command.assert_called_with(
+            "scontrol update NodeName=host-10-18-1-1 State=DRAIN Reason='COBalD/TARDIS'"
+        )
```

Comment:
This is the same test as above (Line 84-86), right? What is the reason to test it twice?